app/views/finders/_results.mustache

```diff
@@ -23,8 +23,14 @@
 
 {{#zero_results}}
   <div class='no-results'>
-    <h2 class='gem-c-heading'>There are no matching results.</h2>
-    <p>Try making your search broader.</p>
+    <p class='govuk-body govuk-!-font-size-19 govuk-!-font-weight-bold'>There are no matching results.</p>
```

Comment:
Rather than putting the same classes on every element, you can put `govuk-!-font-size-19` on the wrapping div.

Reply:
fixed